src/ConnectedMode.UnitTests/Suppressions/ServerIssuesStoreTests.cs

```diff
@@ -33,8 +35,32 @@ public class ServerIssuesStoreTests
         [TestMethod]
         public void MefCtor_CheckIsExported()
         {
-            MefTestHelpers.CheckTypeCanBeImported<ServerIssuesStore, IServerIssuesStore>();
-            MefTestHelpers.CheckTypeCanBeImported<ServerIssuesStore, IServerIssuesStoreWriter>();
+            MefTestHelpers.CheckTypeCanBeImported<ServerIssuesStore, IServerIssuesStore>(
+                MefTestHelpers.CreateExport<ILogger>());
+
+            MefTestHelpers.CheckTypeCanBeImported<ServerIssuesStore, IServerIssuesStoreWriter>(
+                MefTestHelpers.CreateExport<ILogger>());
+        }
+
+        [TestMethod]
+        public void MefCtor_Check_SameInstanceExported()
```

Comment:
Miscellaneous: new test to check that the exports return the same instance rather than different instances.